common/src/main/scala/com/gu/recipeasy/db/DB.scala

```diff
@@ -189,17 +189,43 @@ class DB(contextWrapper: ContextWrapper) {
   }
 
   def progressBarRatio(): Double = {
-    val Key = "verificationCompletionRatio"
 
+    // progressBarRatio ratio is a number between 0 and 1
+
+    def pbr(): java.lang.Double = {
```

Comment:
If you name this what it is i.e `progressBarRatio` then your code is self documenting and you don't need the comment above. In most having to write comments is a code smell. They should only be necessary to explain something that can't be evident from code.